angular/src/lib/data-table/examples/data-table-multi-select-scroll-resize.component.ts

```diff
@@ -96,9 +96,8 @@ export class DataTableMultiSelectCheckboxResizeScrollComponent {
 
   constructor() {
     this.selectedPeople = [
-      {'name': 'Mike', 'state': 'New York', 'age': 45, 'color': 'Orange'},
-      {'name': 'Joe', 'state': 'Texas', 'age': 86, 'color': 'Black'},
-      {'name': 'Thomas', 'state': 'Texas', 'age': 32, 'color': 'Black'}
+      {'name': 'Thomas', 'state': 'Texas', 'age': 32, 'color': 'Black'},
+      {'name': 'John', 'state': 'Texas', 'age': 32, 'color': 'Orange'},
```

Comment:
I noticed that this example was defaulting to selections that did not exist (looks like a copy/paste error from another example).  Anyway, I've removed the bad selections (Mike and Joe) and added another valid one.